react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -218,6 +224,49 @@ class AddPeopleDialog extends Component {
                 });
             });
         }
+
+        return true;
+    }
+
+    /**
+     * Renders the error message if the add doesn't succeed.
+     *
+     * @returns {ReactElement|null}
+     * @private
+     */
+    _renderErrorMessage() {
+        if (!this.state.addToCallError) {
+            return null;
+        }
+
+        const { t } = this.props;
+        const supportLink = interfaceConfig.SUPPORT_URL;
+        const supportLinkContent
+            = supportLink
+            ? ( // eslint-disable-line no-extra-parens
+                <span>
+                    <span>{ `${t('inlineDialogFailure.supportMsg')} `}</span>
+                    <span>
+                        <a
+                            href = { supportLink }
+                            rel = 'noopener noreferrer'
+                            target = '_blank'>
+                            { t('inlineDialogFailure.support') }
+                        </a>
+                    </span>
+                    <span>.</span>
+                </span>
+            )
+            : null;
+
+        return (
+            <div className = 'modal-dialog-form-error'>
+                <InlineMessage
+                    title = { t('addPeople.failedToAdd') }>
+                    { supportLinkContent }
+                </InlineMessage>
```

Comment:
I changed it to always show the message, because this feature is dependant on custom service anyway, so developers should make sure they have some means of communications of problems..